tests/models/jamba/test_modeling_jamba.py

```diff
@@ -697,7 +652,7 @@ def test_simple_batched_generate_with_padding(self):
         output_sentences = self.tokenizer.batch_decode(out)
         self.assertEqual(
             output_sentences[0],
-            "<|startoftext|>Hey how are you doing on this lovely evening? Canyon rins hugaughter glamour Rutgers Singh Hebrew cases Cats",
+            "<|startoftext|>Hey how are you doing on this lovely evening? Canyon rins hugaughter glamour Rutgers Singh<|reserved_797|>cw algunas",
```

Comment:
This might be due to my HW no idea. Would need to see how the slow CI reacts here...

Reply:
yeah no worries on this one